addons/remember-collapsed-categories/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Remember collapsed forum categories",
+  "description": "Categories you collapsed on the forum home page (the list of forums) will stay collapsed when you leave and come back.",
+  "credits": [
+    {
+      "name": "Steve0Greatness",
+      "link": "https://scratch.mit.edu/users/Steve0Greatness/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/discuss/"],
+      "runAtComplete": false
+    }
+  ],
+  "tags": ["community", "forums"],
+  "dynamicDisable": true,
+  "dynamicEnable": true,
```

Comment:
I don't think this addon supports dynamicDisable. But because of the nature of this addon, it's hard to tell what supporting dynamicDisable would mean 🤔